packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion/tag.ts

```diff
@@ -0,0 +1,48 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `tag` variant for component `suggestion`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function suggestionTag(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    tag: {
+      minHeight: theme('spacing.32'),
+      paddingTop: 0,
+      paddingBottom: 0,
+      alignItems: 'center',
+      paddingInlineStart: theme('spacing.8'),
+      paddingInlineEnd: theme('spacing.8'),
+
+      backgroundColor: theme('colors.neutral.0'),
+      borderWidth: theme('borderWidth.1'),
+      borderColor: 'currentColor',
+
+      '&:hover': {
+        backgroundColor: theme('colors.neutral.10'),
+        textDecoration: 'none'
+      },
+
+      '&.suggestion-md': {
```

Comment:
I think that this will cause the problem that Ivan said, if you use sugggestion-md, the styles related with tag will be loaded even if you are not using them

Reply:
@tajespasarela said that this could be solved using variables or something. For the moment, we will leave it as it is.